docs/UserGuide.md

```diff
@@ -133,8 +135,55 @@ Examples:
 * `list c` followed by `view c 1` views the first contact in the companies list:
   ![show the information of the first company](images/viewCompany.png)
 
-Thank you for pointing that out. I hope this revised section aligns better with your application's functionality.
+### Finding a person by name or tag: 'find p'
 
+Find a person by name or tag.
+
+Format: `find p [n/KEYWORD] [t/TAG]`
+
+Search Criteria:
+- The search is case-insensitive. e.g. `john` will match `John`.
```

Comment:
Another nit here. Could we use `*` for bullet points instead of `-` so it aligns with the other commands in the UG?